hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -66,20 +67,186 @@ tests:
+          # still using the 10 minutes max
+          - exp_annotations:
+              description: "unsafe modification for the networkaddonsconfigs.kubevirt.io/jsonpatch annotation in the HyperConverged resource."
+              summary: "1 unsafe modifications were detected in the HyperConverged resource."
+            exp_labels:
+              severity: "info"
+              annotation_name: "networkaddonsconfigs.kubevirt.io/jsonpatch"
+
+      - eval_time: 12m
+        alertname: KubevirtHyperconvergedClusterOperatorUSModification
+        exp_alerts:
+          - exp_annotations:
+              description: "unsafe modification for the kubevirt.kubevirt.io/jsonpatch annotation in the HyperConverged resource."
+              summary: "3 unsafe modifications were detected in the HyperConverged resource."
+            exp_labels:
+              severity: "info"
+              annotation_name: "kubevirt.kubevirt.io/jsonpatch"
+          - exp_annotations:
+              description: "unsafe modification for the containerizeddataimporter.kubevirt.io/jsonpatch annotation in the HyperConverged resource."
+              summary: "3 unsafe modifications were detected in the HyperConverged resource."
+            exp_labels:
+              severity: "info"
+              annotation_name: "containerizeddataimporter.kubevirt.io/jsonpatch"
+          # still using the 10 minutes max
+          - exp_annotations:
+              description: "unsafe modification for the networkaddonsconfigs.kubevirt.io/jsonpatch annotation in the HyperConverged resource."
+              summary: "1 unsafe modifications were detected in the HyperConverged resource."
+            exp_labels:
+              severity: "info"
+              annotation_name: "networkaddonsconfigs.kubevirt.io/jsonpatch"
+
+      # no alert if the value is 0
+      - eval_time: 13m
+        alertname: KubevirtHyperconvergedClusterOperatorUSModification
+        exp_alerts:
+          - exp_annotations:
+              description: "unsafe modification for the kubevirt.kubevirt.io/jsonpatch annotation in the HyperConverged resource."
+              summary: "3 unsafe modifications were detected in the HyperConverged resource."
+            exp_labels:
+              severity: "info"
+              annotation_name: "kubevirt.kubevirt.io/jsonpatch"
+          - exp_annotations:
+              description: "unsafe modification for the containerizeddataimporter.kubevirt.io/jsonpatch annotation in the HyperConverged resource."
+              summary: "3 unsafe modifications were detected in the HyperConverged resource."
+            exp_labels:
+              severity: "info"
+              annotation_name: "containerizeddataimporter.kubevirt.io/jsonpatch"
+
+      # no alert if the value is 0
+      - eval_time: 14m
+        alertname: KubevirtHyperconvergedClusterOperatorUSModification
+        exp_alerts: []
+
+      # Still using the max in the last 10 minutes
```

Comment:
You removed 10 minutes window from the query so we don't need these tests anymore.